### PR TITLE
Rte disable

### LINKED ISSUE #266: React-admin-rte - option to disable ControlButtons

### PR DESCRIPTION
**Pull request by rabengraph**
resolves #266
depends on #273

### CODE REVIEW

**Review by nsams — APPROVED** [review]
build is failing, but that doesn't seem to be related to your changes

**Comment by nsams** [comment]
The input area could get a styling similar to an disabled input (gray background and gray text)

the difference is visible here https://deploy-preview-285--comet-admin.netlify.app/iframe.html?id=comet-admin-rte-field--field

**Review by dkarnutsch — APPROVED** [review]

**Comment by rabengraph** [comment]
> The input area could get a styling similar to an disabled input (gray background and gray text)
> 
> the difference is visible here https://deploy-preview-285--comet-admin.netlify.app/iframe.html?id=comet-admin-rte-field--field

I changed the text-color to the theme's disabled-text-color.
The background-color for disabled text-fields is not gray either, so I kept this for the rte as well.
https://deploy-preview-285--comet-admin.netlify.app/iframe.html?id=comet-admin-rte-field--field

**Review by nsams — APPROVED** [review]

File: packages/admin-rte/src/core/Rte.sc.ts [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 export const Root = styled.div`
     border: 1px solid ${({ theme }) => theme.rte.colors.border};
 `;
 
-export const EditorWrapper = styled.div`
+export const EditorWrapper = styled.div<{ disabled?: boolean }>`
+    ${({ disabled, theme }) =>
+        disabled &&
+        css`
+            color: ${theme.palette.text.disabled} !important;
```
Comment (johnnyomair) [inline packages/admin-rte/src/core/Rte.sc.ts:11]: You could remove the `!important` Flag by increasing the specificity: https://styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity

**Review by johnnyomair — APPROVED** [review]

**Review by jamesricky — APPROVED** [review]

**Pull request merged**